addon/stories/pix-multi-select.stories.js

```diff
@@ -101,47 +106,83 @@ Examples de valeur préselectionné:
 | ------------- |:-------------:|:-------------------:|:----------:|----------:|
 | id            | string        |                     |            |     non   |
 | title         | string        |                     |            |     non   |
-| onSelect      | function      |                     |            |     non   |
-| emptyMessage  | string        |                     |            |     oui   |
-| selected      | array         |                     |            |     oui   |
+| emptyMessage  | string        |                     |            |     non   |
+| placeholder   | string        |                     |            |     non   |
 | options       | array         |                     |            |     non   |
+| onSelect      | function      |                     |            |     non   |
+| selected      | array         |                     |    []      |     oui   |
+| displayOnFocus| boolean       |                     |    false   |     oui   |
+| isSearchable  | boolean       |                     |    false   |     oui   |
 `;
 
 export const multiSelect = () => {
   return {
     template: canvasContent,
     context: {
-      optionsNoScroll : [
-        { value: '1', label: 'tomate' },
-        { value: '2', label: 'fromage' },
-        { value: '3', label: 'gruyère' },
-        { value: '4', label: 'olive' },
-        
-      ],
-      options: [
-        { value: '1', label: 'Tomate' },
-        { value: '2', label: 'Fromage' },
-        { value: '3', label: 'Gruyère' },
-        { value: '4', label: 'Olive' },
-        { value: '5', label: 'Jambon' },
-        { value: '6', label: 'Tapenade' },
-        { value: '7', label: 'Tzatziki' },
-      ],
+      idStars: 'stars',
+      titleStars: 'Sélectionner le niveau souhaité',
       optionsStars: [
         { value: '1', total : 7 },
         { value: '2', total : 7 },
         { value: '3', total : 7 },
```

Comment:
```suggestion
        { value: '1', total : 3 },
        { value: '2', total : 3 },
        { value: '3', total : 3 },
```
C'est perturbant sinon, on pense que le scroll fonctionne pas